src/Apps/IdentityVerification/IdentityVerificationApp/StartIdentityVerification.tsx

```diff
@@ -0,0 +1,168 @@
+    internalID: string
+  }
+}
+
+export const StartIdentityVerification: React.FC<Props> = ({
+  identityVerification,
+  relay: { environment },
+}) => {
+  const [requesting, setRequesting] = useState(false)
+  const [showErrorModal, setShowErrorModal] = useState(false)
+  const { trackEvent } = useTracking()
+
+  const trackClickedContinueToVerification = () => {
+    trackEvent({
+      context_page_owner_id: identityVerification.internalID,
+      action_type: Schema.ActionType.ClickedContinueToIdVerification,
+      context_page: Schema.PageName.IdentityVerificationPage,
+    })
+  }
+
+  async function startIdentityVerification() {
+    const mutation = new Promise<string>((resolve, reject) => {
+      commitMutation<StartIdentityVerificationMutation>(environment, {
+        mutation: graphql`
+          mutation StartIdentityVerificationMutation(
+            $input: startIdentityVerificationMutationInput!
+          ) {
+            startIdentityVerification(input: $input) {
+              startIdentityVerificationResponseOrError {
+                ... on StartIdentityVerificationSuccess {
+                  identityVerificationFlowUrl
+                }
+                ... on StartIdentityVerificationFailure {
+                  mutationError {
+                    detail
+                    error
+                    message
+                  }
+                }
+              }
+            }
+          }
+        `,
+        variables: {
+          input: { identityVerificationId: identityVerification.internalID },
+        },
+        onError: reject,
+        onCompleted: (response, errors) => {
+          if (errors && errors.length > 0) {
+            reject(new Error(JSON.stringify(errors)))
+          } else {
+            const {
+              startIdentityVerification: {
+                startIdentityVerificationResponseOrError: {
+                  identityVerificationFlowUrl,
+                  mutationError,
+                },
+              },
+            } = response
+            if (mutationError) {
+              reject(new Error(JSON.stringify(mutationError)))
+            } else {
+              resolve(identityVerificationFlowUrl)
+            }
+          }
+        },
+      })
+    })
+
+    mutation
+      .then(handleMutationSuccess)
+      .catch(handleMutationError)
+      .finally(() => {
+        setRequesting(false)
+      })
+  }
+
+  const handleMutationSuccess = (identityVerificationFlowUrl: string) => {
+    location.assign(identityVerificationFlowUrl)
+  }
+
+  const handleMutationError = (error: Error) => {
+    logger.error("Error when trying to start identity verification", error)
+    setShowErrorModal(true)
+  }
+  return (
+    <>
+      <ErrorModal
+        show={showErrorModal}
+        contactEmail="verification@artsy.net"
+        onClose={() => {
+          setShowErrorModal(false)
+        }}
+      />
+      <Box px={[2, 3]} mb={6} mt={4}>
+        <Box
+          mx={["auto"]}
```

Comment:
Similar to @yuki24's [comment above](https://github.com/artsy/reaction/pull/3294/files#r396483819), I wonder if `mx="auto"` is sufficient here.